src/CommunityVoices/App/Website/View/Quote.php

```diff
@@ -283,9 +283,13 @@ public function getQuoteUpload($request, $errors = self::ERRORS_DEFAULT)
                 '<form>' . $this->transcriber->toXml($form) . '</form>'
             );
         }
-        $repeatedQuote = $request->request->has('submit_more') && empty($errors->upload->errors);
-        $repeatedQuoteXMLElement = new SimpleXMLElement(
-            $this->transcriber->toXml(['repeatedQuote' => [$repeatedQuote]])
+        
+        $repeatedQuoteErrorFree = NULL;
```

Comment:
Absolutely not an issue, but I usually write null lower-case.  Just something for code consistency.